week1/week1-1/css/style.css

```
@@ -0,0 +1,314 @@
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 5rem;
+  background-color: #808080;
+  width: 100%;
+  height: 10rem;
+}
+.routing__nav .nav__ul {
+  display: flex;
+  gap: 5rem;
+}
+.routing__nav .nav__ul .nav__li a {
+  color: white;
+  font-size: 2rem;
+}
+.routing__nav .nav__ul .nav__li a:hover {
+  text-decoration: underline;
+  color: purple;
+}
+
+.content__header {
+  display: flex;
+  position: sticky;
+  top: 0;
+  align-items: center;
+  justify-content: center;
+  margin-top: 3rem;
+  background: radial-gradient(circle, #eeaeca 0%, rgb(148, 187, 233) 100%);
+  width: 150rem;
+  height: 4rem;
+  color: purple;
+}
+
+.content__section {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 3rem;
+  border: 3px solid #a0a0a0;
+  padding-right: 2rem;
+  padding-left: 2rem;
+  max-width: 140rem;
+  height: -moz-fit-content;
+  height: fit-content;
+}
+.content__section .content__ul {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  justify-content: center;
+}
+.content__section .content__ul .content__li {
+  display: flex;
+  flex-basis: 26rem;
+  flex-grow: 1;
+  flex-shrink: 1;
+  justify-content: center;
+}
+.content__section .content__ul .content__li .content__img {
+  width: 100%;
+  height: 100%;
+}
+
+.content2__section {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  justify-content: center;
+  margin-top: 3rem;
+  margin-bottom: 3rem;
+  border: 3px solid #a0a0a0;
+  width: 100%;
+}
+.content2__section .main__article {
+  width: 600rem;
+}
+.content2__section .main__article img {
+  width: 100%;
+  height: 100%;
+}
+.content2__section .content2__ul {
+  display: flex;
+  flex-grow: 1;
+  flex-wrap: wrap;
+  gap: 1rem;
+  justify-content: center;
+}
+.content2__section .content2__ul .content2__li {
+  display: flex;
+  flex-basis: 35rem;
+  flex-grow: 1;
+  flex-shrink: 1;
+  justify-content: center;
+}
+.content2__section .content2__ul .content2__li .content2__img {
+  width: 100%;
+  height: 100%;
+}
+
+@media (max-width: 1520px) {
+  .content2__section {
+    display: flex;
+    flex-direction: column;
+  }
+  .content2__section .main__article {
+    width: 200rem;
+    height: 20rem;
+  }
+  .content2__section .main__article .content__img {
+    width: 100%;
+    height: 100%;
+  }
+}
+@media (max-width: 1075px) {
+  .content2__section .content2__ul .content2__li {
+    flex-basis: 30rem;
+  }
+}
+@media (max-width: 941px) {
+  .content2__section .content2__ul .content2__li {
+    flex-basis: 24rem;
+  }
+}
+@media (max-width: 775px) {
+  .content2__section .content2__ul .content2__li {
+    flex-basis: 22.5rem;
+  }
+}
+@media (max-width: 715px) {
+  .content2__section .content2__ul .content2__li {
+    flex-basis: 18.5rem;
+  }
+}
+@media (max-width: 595px) {
+  .content2__section .content2__ul .content2__li {
+    flex-basis: 14rem;
+  }
+}
+.content3-1__section {
+  display: grid;
+  grid-template-rows: 20rem 20rem 20rem 20rem;
```

```suggestion
  grid-template-rows: repeat(4, 20rem);
```
이렇게 더 간단하게 만들 수 있답니당!